components/shared/Loader.vue

```diff
@@ -24,62 +24,61 @@
   </NeoLoading>
 </template>
 
-<script lang="ts">
-import { Component, Prop, Vue, Watch } from 'nuxt-property-decorator'
+<script lang="ts" setup>
 import { randomIntegerBetween } from '@/utils/calculation'
 import { NeoLoading } from '@kodadot1/brick'
 
-@Component({ components: { NeoLoading } })
-export default class Loader extends Vue {
-  @Prop(String) public status!: string
-  @Prop(Boolean) public value!: boolean
-  @Prop({ type: Boolean, default: true }) public canCancel
+const emit = defineEmits(['input'])
+const props = withDefaults(
+  defineProps<{
+    status: string
+    value: boolean
+    canCancel: boolean
+  }>(),
+  {
+    canCancel: true,
+  }
+)
+
+const { $i18n } = useNuxtApp()
+const placeholder = ref('/preloader.svg')
```

Comment:
It's unnecessary to wrap in ref.
```suggestion
const placeholder = '/preloader.svg'
```